app/vep/models/pipeline_model.py

```diff
@@ -23,16 +22,26 @@ class VEPConfigParams(BaseModel):
     vcf: FilePath
     vep_config: FilePath
     outdir: DirectoryPath
+    bin_size: int = 3000
+    sort: bool = True
+    output_prefix: str = ""
 
     @model_serializer
     def vep_config_serialiser(self):
-        vcf_str = '"vcf":"' + self.vcf.as_posix() + '"'
-        config_str = '"vep_config":"' + self.vep_config.as_posix() + '"'
-        outdir_str = '"outdir":"' + self.outdir.as_posix() + '"'
-        stringified_encoded_json = (
-            "{" + vcf_str + "," + config_str + "," + outdir_str + "}"
+        vcf_str = f'"input": "{self.vcf.as_posix()}"'
+        config_str = f'"vep_config": "{self.vep_config.as_posix()}"'
+        outdir_str = f'"outdir": "{self.outdir.as_posix()}"'
+        bin_str = f'"bin_size": {self.bin_size}'
+        sort_str = f'"sort": {"true" if self.sort else "false"}'
+        prefix_str = (
+            f'"output_prefix": "{self.output_prefix}"' if self.output_prefix else ""
         )
-        return stringified_encoded_json
+        json_str = (
+            "{" + ", ".join(
+                [vcf_str, config_str, outdir_str, bin_str, sort_str, prefix_str]
+            ) + "}"
+        )
+        return json_str
```

Comment:
Did you test the file being generated here and see if it actually works with the VEP ?
Alternatively, did you compare it with the current working file.

Reply:
I could not reproduce it when running ToolsAPI on my local machine (got job ID back when launching the workflow with payload from PR #22). Where does this 400 come from?

Reply:
Edit: I could reproduce the error when changing the serializer output, e.g. adding another json.dumps() around it. So the current code should work fine.

Reply:
This works required config update